doc/migration/version-upgrade-guide.md

```diff
@@ -940,6 +940,10 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 
```

Comment:
@nikhil-thomas, 

The release was made. Then, is required to rebase with the master and just your sections/content will be added at the and : 

```
## Unreleased (Master branch)

### Breaking changes

#### In the test-framwork

The methods `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()` was added pkg/test in order to replace `ctx.GetNamespace()` which is deprecated. In this way replace the use of `ctx.GetNamespace()` in your project with `ctx.GetOperatorNamespace()`

### Breaking Changes on Commands

This release contains breaking changes in some commands.

- The --namespace flag from `operator-sdk run --local` command, `operator-sdk test --local` command and `operator-sdk cleanup` command was deprecated and was replaced by --watch-namespace and --operator-namespace .

Note that --watch-namespace can be used to set the namespace(s) which the operator will watch for changes. It will set the environment variable WATCH_NAMESPACE. Use explicitly an empty string to watch all namespaces or inform a List of namespaces such as "ns1,ns2" when the operator is cluster-scoped. If you use a List, then it needs contains the namespace where the operator is "deployed" in since the default metrics implementation will manage resources in the Operator's namespace. By default, it will be the Operator Namespace.

Then, use the flag --operator-namespace to inform the namespace where the Operator will be "deployed" in and then, it will set the environment variable OPERATOR_NAMESPACE. If this value is not set, then it will be the namespace defined as default in the Kubeconfig.
``` 

Note that just the above is required. The lines https://github.com/operator-framework/operator-sdk/pull/2617/files#diff-59e439e5373edfb5478c1999335f4842R990-R1001 can be removed.

Reply:
done

Reply:
See that your content was NOT removed from 0.16x section yet : https://github.com/operator-framework/operator-sdk/blob/56364ef881f6be5c077f3d9f424ce9d4fc192fc6/doc/migration/version-upgrade-guide.md#breaking-changes-on-commands-1

Reply:
https://github.com/operator-framework/operator-sdk/pull/2617/files#diff-59e439e5373edfb5478c1999335f4842R980